dump/dump_test.go

```diff
@@ -9,6 +9,9 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/siddontang/go-log/log"
+	"github.com/siddontang/go-mysql/mysql"
+
```

Comment:
please remove this line

Reply:
these two packages are used in unit test